include/nbl/video/alloc/StreamingGPUBufferAllocator.h

```diff
@@ -14,18 +14,23 @@ namespace video
 
 class StreamingGPUBufferAllocator : protected SimpleGPUBufferAllocator
 {
+    private:
+        void* mapWrapper(IDriverMemoryAllocation* mem, IDriverMemoryAllocation::E_MAPPING_CPU_ACCESS_FLAG access, const IDriverMemoryAllocation::MemoryRange& range) noexcept;
+        void unmapWrapper(IDriverMemoryAllocation* mem) noexcept;
```

Comment:
you have to make a `.cpp` file for the wrapper's definitions anyway, so dont make wrappers and just move the methods that require them to the said `.cpp`

Reply:
wrappers make it more DRY

Reply:
dont see how its more dry, each wrapper is used.. once (there's one that gets used twice)

and how is it more dry than just moving the old code to the cpp?